metacatalog_corr/models.py

```diff
@@ -240,19 +242,29 @@ def create(
         if len(left) == 0 or len(right) == 0:
             return None
 
-        # loaded data is a list of lists [[x], [y], [z]] -> unstack
+        # loaded data is a list of lists [[x], [y], [z]] -> stack
         left = np.hstack(left)
         right = np.hstack(right)
 
-        # calculate
-        corr = metric.calc(left, right)
+        # calculate, if warnings occur, store them in the matrix
+        with warnings.catch_warnings(record=True) as w:
+            corr = metric.calc(left, right)
+
+            if w:
+                warning = []
+                for warn in w:
+                    warning.append(f'{warn.category.__name__}: {str(warn.message)}')
```

Comment:
With the changes suggested above, we can put the category into a column of it's own. That would make them in principle searchable.